### ISSUE TITLE
MR flag: remove source branch after merge

### ISSUE THREAD

**Issue by doits** [zaquestion/lab#194]
Would be nice to have the possibility to check "remove source branch after merge" when creating a new MR through lab.

I suppose this is the case for a new flag for `lab merge-request`? Or how would you implement this? (I could start to do so if I knew how it should be done.)

**Comment by zaquestion** [comment]
Yeah I believe it's just an option in `CreatMergeRequestOptions`, look in the `cmd/mr_create.go` file.

**Comment by doits** [comment]
> Yeah I believe it's just an option in CreatMergeRequestOptions, look in the cmd/mr_create.go file.

It should be :-)  see https://github.com/xanzy/go-gitlab/pull/439
